File: src/main/java/com/yupi/template/model/vo/ModelPricingVO.java
package com.yupi.template.model.vo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;
import java.math.BigDecimal;

/**
 * 模型价格视图对象，用于缓存及成本计算
 *
 * @author <a href="https://codefather.cn">编程导航学习圈</a>
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ModelPricingVO implements Serializable {

    private BigDecimal inputPrice;

    private BigDecimal outputPrice;

    private static final long serialVersionUID = 1L;
}

File: go-backend/internal/model/vo/model_pricing_vo.go
// Package vo 模型价格视图对象
// @author <a href="https://codefather.cn">编程导航学习圈</a>
package vo

type ModelPricingVO struct {
	InputPrice  float64 `json:"inputPrice"`
	OutputPrice float64 `json:"outputPrice"`
}
